client/app/house/components/RemoveMemberAlertModal.tsx

```diff
@@ -0,0 +1,71 @@
+import {
+  AlertDialog,
+  AlertDialogAction,
+  AlertDialogCancel,
+  AlertDialogContent,
+  AlertDialogDescription,
+  AlertDialogFooter,
+  AlertDialogHeader,
+  AlertDialogTitle,
+} from '@/_components/ui'
+import { HouseType } from '@/_types'
+import { useRemoveMember } from '../hooks'
+
+type RemoveMemberAlertModalProps = {
+  house: HouseType
+  isOpenAlert: boolean
+  setIsOpenAlert: (isOpenAlert: boolean) => void
+  setIsOpenModal: (isOpenModal: boolean) => void
+  removedMemberId: number | null
+}
+
+const RemoveMemberAlertModal = ({
```

Comment:
The scroll bar shows when you cancel and back to the member modal, could you fix it? Normally, when you shows a modal, there should be no scroll bar 
https://github.com/HAK2024/fairy-share/assets/66394413/171bc72e-bdd9-464d-8f11-7f0f1852909e

Reply:
I fixed it by making some intervals
https://github.com/HAK2024/fairy-share/pull/93/commits/030c83e516289c6613a7d9efe878b216b7f00424